app/views/main.scala.html

```diff
@@ -41,6 +41,10 @@
             partialRetriever: uk.gov.hmrc.play.partials.PartialRetriever
 )
 
+@serviceSuffix = @{
+    s"- ${TaiConstants.SERVICE_NAME_PUBLIC} - GOV.UK"
```

Comment:
Any reason why this is different to the nav title? Can they both use the same config?

Reply:
As discussed - have updated to use Messages("tai.service.navTitle") as suggested, and removed the new constant I introduced (was based upon the labelling of the PTA tile that brings the user into our service).